css/main.css

```diff
@@ -0,0 +1,311 @@
+  font-size: 14px;
+  font-weight: 600;
+  text-transform: uppercase;
+}
+/* End Header */
+
+/* Hero Block */
+#hero {
+  background-image: url('../img/hero-image 1.png');
+  background-size: auto 100%;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 0;
+  padding-top: 27.64%; /* (img-width / img-height * container-width) */
+  position:relative;
+  margin-bottom: 60px;
+  min-height: 50px;
+}
+
+#hero .hero-content {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  margin-left: 5%;
+  justify-content: center;
+}
+
+#hero .hero-title {
+  font-size: 42px;
+  font-weight: 600;
+  line-height: 51px;
+  color: #082264;
+  margin-bottom: 25px;
+}
+
+#hero .hero-text {
+  font-size: 18px;
+  line-height: 22px;
+  color: #082264;
+}
+/* End Hero Block */
+
+/* Section */
+.main-content {
+  width: 90%;
+  margin: 0 5%;
+}
+
+section {
+  width: 100%;
+}
+
+section article {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 60px;
+}
+
+section article:nth-child(odd) {
+  flex-direction: row;
+}
+
+section article:nth-child(even) {
+  flex-direction: row-reverse;
+}
+
+section article .article-content {
+  max-width: 50%;
+}
+
+section article .article-table-content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+}
+
+section article .article-title {
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 22px;
+  color: #55B8F0;
+  margin-bottom: 8px;
+}
+
+section article .article-image {
+  display: flex;
+  width: 45%;
+}
+
+section article .article-image img {
+  width: 100%;
+  height: auto;
+}
+
+section article .article-subtitle {
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 44px;
+  color: #082264;
+  margin-bottom: 14px;
+}
+
+section article .article-text {
+  font-size: 18px;
+  line-height: 22px;
+  color: #082264;
+}
+
+section article .article-button {
+  background: #55B8F0;
+  border-radius: 30px;
+  width: 216px;
+  height: 40px;
+  line-height: 40px;
+  margin-top: 30px;
+
+  font-weight: 600;
+  font-size: 16px;
+  text-align: center;
+  color: #FFFFFF;
+}
+/* End Section */
+
+/* Footer */
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background: #082264;
+  height: 80px;
+  width: 100%;
+  text-align: center;
+  color: #FFFFFF;
+}
+
+footer a {
+  text-decoration: underline;
+  color: #FFFFFF;
+}
+/* End Footer */
+
+.chevron {
+  color: #A4A3AD;
+  cursor: pointer;
+  margin-right: 15px;
+}
+
+.chevron.chevron-active {
+  color: #5C5C6C;
+}
+
+.chevron-down::before {
+  border-style: solid;
+  border-width: 0.15em 0.15em 0 0;
+  content: '';
+  display: inline-block;
+  height: 0.45em;
+  left: 0.15em;
+  position: relative;
+  top: 0.15em;
+  transform: rotate(135deg);
+  vertical-align: top;
+  width: 0.45em;
+  margin-left: 5px;
+}
+
+
+/* Flex Table */
+.flex-table {
+  display: flex;
```

Comment:
Using flex for table in this case isn't recommended.
Adding text without spaces will cause content overlapping (cells won't fit to content).